odoo/addons/base/models/res_lang.py

```diff
@@ -159,6 +159,17 @@ def _activate_lang(self, code):
             lang.active = True
         return lang
 
+    def _activate_and_update_lang(self, code):
```

Comment:
The purpose is to install the language if not installed yet.  I suggest to rename it `_activate_and_install_lang`.